src/frameworks/vue/template/vue.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  css: {
```

Comment:
Do we need to reference our stylelint rules here?

Reply:
same goes for eslint

Reply:
lint isnt configured through the vueconfig, thats what the eslint & stylelint files are for